SECURITY.md

```diff
@@ -0,0 +1,12 @@
+# Security Policy
+
+## Supported Versions
+
+
+| Version | Supported          |
+| ------- | ------------------ |
+| [0.0.2.4](https://ticketsbot.net/)  | :white_check_mark: |
```

Comment:
Why link to ticketsbot?

Reply:
> Why link to ticketsbot?

Nothing else in mind to put atm, Just a *something*

Reply:
If preferred, you would be able to change/remove it, as there is no "official" supported/unsupported version/s of the bot.